web/packages/teleterm/src/ui/DocumentAccessRequests/useAccessRequests.tsx

```diff
@@ -104,6 +104,8 @@ export function makeUiAccessRequest(request: TshdAccessRequest) {
     suggestedReviewers: request.suggestedReviewers,
     thresholdNames: request.thresholdNames,
     resources: request.resources,
+    reasonMode: request.reasonMode || 'optional',
+    reasonPrompts: request.reasonPrompts || [],
```

Comment:
This comes from a gRPC message, so it's an empty array by default.
```suggestion
    reasonPrompts: request.reasonPrompts
```